src/urbs.jl

```diff
@@ -108,19 +127,35 @@ function build_model(filename, debug=false)
 	#build model
 	m = Model()
 
-	@defVar(m, cost[timeseries] >= 0)
+	@defVar(m, cost >=0)
 	@defVar(m, production[timeseries, numprocess] >= 0)
-	@setObjective(m, Min, sum{cost[t], t = timeseries})
-
-	@addConstraint(m, determine_cost[t = timeseries],
-	               cost[t] == sum{production[t, p] * processes[p].cost,
-	                              p = numprocess})
-
-	@addConstraint(m, check_max_prod[t = timeseries, p = numprocess],
-	               production[t,p] <= processes[p].max_prod)
-	@addConstraint(m, check_min_prod[t = timeseries, p = numprocess],
-	               production[t,p] >= processes[p].min_prod)
-
+	@defVar(m, cap_avail[numprocess] >= 0)
+	@setObjective(m, Min, cost)
+
+	# cost constraints
+	@addConstraint(m, cost ==
+	               # all commodity costs
+				   sum{production[t, p] * processes[p].cost_com,
+				       t = timeseries, p = numprocess} +
+				   # investment costs
+				   sum{(cap_avail[p]-processes[p].cap_init) *
+				       processes[p].cost_inv, p = numprocess} +
+				   # fix costs
+				   sum{cap_avail[p] * processes[p].cost_fix, p = numprocess} +
+				   # variable costs
+				   sum{production[t,p] * processes[p].cost_var,
+				       t = timeseries, p = numprocess})
+
+	# capacity constraints
+	# assume that cap_inst <= cap_min
```

Comment:
Can we assume that the initial capacity of a process is <= the minimal capacity?
If not, can capacity be reduced and at what price?